Source/w_wad.c

```diff
@@ -105,25 +105,26 @@ char *AddDefaultExtension(char *path, const char *ext)
 void NormalizeSlashes(char *str)
 {
   char *p;
-
+/*
```

Comment:
Please keep blank lines intact and just add the comment marker lines. Also, please add a comment why we disable this code path.